SalesforceFoundation/HEDAP
# Fixing reparenting affiliation creating duplicate affiliation

# Critical Changes

# Changes
We've fixed a bug that created duplicate Affiliations when users reparented an existing Primary Affiliation with a different Contact/Account.

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
1. Deploy HEDA to a scratch org
2. Create a non-admin account(business org etc)
3. Create two contacts
4. Set the primary field(primary business org for instance) as contact#1
5. Go to contact#1 and find the new affiliation(affiliation#1) under related list(affiliation account)
6. Go to affiliation record page and update the parent contact to contact#2
7. Make sure no new affiliation is created, affiliation#1 is under contact#2 and is primary.
